www/site/content/docs/chef-workstation/chef-workstation-app.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Chef Workstation App"
+[menu]
+  [menu.docs]
+    parent = "Chef Workstation"
+    weight = "100"
++++
+
+# About Chef Workstation App
+
+The Chef Workstation App (CWA) is an early-release desktop application that provides additional services for Chef Workstation:
+
+* Update checking and notifications
+* Chef Workstation version information
+
+Additional features and integrations will be rolled out in future updates.
+
+## Running the Chef Workstation App
+
+### Windows
+
+Start Chef Workstation App from "Recently Added" shortcuts in the Start menu, or the Chef Workstation folder.
+
+### Linux
+Chef Workstation App requires a running display manager with support for system tray icons. On some distributions you may also need to install the libraries  `TODO` and `TODO`
+
+Start Chef Workstation App by running the command `chef-workstation-app`.
+
+### Mac
+Start `Chef Workstation App` from your Applications folder.
+
+## Disabling Automatic Update Checks
+
+To disable automatic update checking of  the update channel of CWA, add or modify the `enable` setting under `updates` in [config.toml](TODO: link):
+
+```
+[updates]
+enable=false
+```
+
+## Setting update channel
+
+To set the update channel used for update checking, bring up the CWA tray app menu (Windows/Linux) or the application menu (Mac) and select "About Chef Workstation".
+
+Select the "Channel" button and choose your preferred channel.  This will trigger an immediate update check.
+
+TODO: screen captures of controlling in-app
```

Comment:
Do we have a need for screenshots here? I think the text is pretty straightforward .. thoughts?

Reply:
I'm alright without them.  It is pretty straightforward.

Reply:
Is the icon the same on all systems? If so, we could use that. It is nice to know what to look for.

Reply:
yes, the icon is the same.